zbus/Cargo.toml

```diff
@@ -35,8 +35,9 @@ async-io = [
   "futures-util/io",
 ]
 tokio = ["dep:tokio"]
-vsock = ["dep:vsock", "dep:async-io"]
-tokio-vsock = ["dep:tokio-vsock", "tokio"]
+vsock = ["dep:vsock", "dep:async-io", "vsock_address"]
+tokio-vsock = ["dep:tokio-vsock", "tokio", "vsock_address"]
+vsock_address = []
```

Comment:
btw, the chances of anyone **directly** enabling this feature w/o the `vsock` or `tokio-vsock`  features are extremely slim IMO. Given we've plenty of features already, I'd strongly suggest we drop this.